backend/src/Tabulators/AllocatedScore.ts

```diff
@@ -0,0 +1,369 @@
+import { ballot, candidate, fiveStarCount, allocatedScoreResults, allocatedScoreSummaryData, summaryData, totalScore } from "./ITabulators";
+
+import { IparsedData } from './ParseData'
+const ParseData = require("./ParseData");
+declare namespace Intl {
+    class ListFormat {
+        constructor(locales?: string | string[], options?: {});
+        public format: (items: string[]) => string;
+    }
+}
+// converts list of strings to string with correct grammar ([a,b,c] => 'a, b, and c')
+const formatter = new Intl.ListFormat('en', { style: 'long', type: 'conjunction' });
```

Comment:
Is this variable used? The logic seems like a front end problem, so I was surprised to see it here

Reply:
It was accidently copied over from the STAR tabulator which uses it to write human readable logs. It could possibly be used in all the tabulators but removed it for now.